Lib/base64.py

```diff
@@ -28,6 +29,7 @@
 
 _translation = [chr(_x) for _x in range(256)]
 EMPTYSTRING = ''
+warnpy3k_with_fix("base64.EMPTYSTRING is not supported in 3.x", "introduce an alias", stacklevel=2)
```

Comment:
I'm not sure what "introduce an alias" means.

Reply:
implement/create that property yourself just like how we manually introduce missing features/methods?

Reply:
Do you mean like, import the new name an alias with `import ... as`?

Reply:
This property doesnt exist in Py3.x, so for neutral support, there needs to be some mimicking. We had something like this with cmp, where folks had to implement the missing method, if they didnt want to change to the rich comparison operations.

Reply:
The logic seems to be the same as cmp, so I am inclined to agree